Anchor.toml

```diff
@@ -12,10 +12,10 @@ url = "https://api.apr.dev"
 
```

Comment:
Since anchor.toml is in `.gitignore` should we delete it in this PR

Reply:
hmm I added it because it was adding my Anchor.toml changes to my commits

Reply:
Yeah good point